README.md

```diff
@@ -2,6 +2,11 @@
 
 # Kubeapps Installer
 
+Kubeapps Installer is a [Bitnami](https://bitnami.com) tool to super-charge your Kubernetes cluster with:
+ * Your own app [dashboard](https://kubeapps.com/), allowing you to deploy Kubernetes-ready apps into your cluster with a single click.
+ * [Kubeless](http://kubeless.io/) - a Kubernetes-native Serverless Framework, compatible with [serverless.com](https://serverless.com).
```

Comment:
I am not sure if we want to mention serverless.com

Reply:
I thought to mentioned to give it a bit more of interest, but I am fine either. I think probably is a question for @sebgoa